test/new-e2e/tests/containers/docker_test.go

```diff
@@ -0,0 +1,145 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package containers
+
+import (
+	"context"
+	"fmt"
+	fakeintake "github.com/DataDog/datadog-agent/test/fakeintake/client"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/runner"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/infra"
+	dockervm "github.com/DataDog/test-infra-definitions/scenarios/aws/dockerVM"
+	"github.com/pulumi/pulumi/sdk/v3/go/auto"
+	"github.com/stretchr/testify/suite"
+	"os"
+	"testing"
+	"time"
+)
+
+type DockerSuite struct {
+	baseSuite
+	fullAgentImagePath string
+	stackName          string
+}
+
+func TestDockerSuite(t *testing.T) {
+
+	agentFullImage := os.Getenv("AGENT_FULL_IMAGE")
+
+	stackName := "docker-stack"
+
+	if agentFullImage != "" {
+		suite.Run(t, &DockerSuite{stackName: stackName, fullAgentImagePath: agentFullImage})
+	} else {
+		// Full Agent
+		suite.Run(t, &DockerSuite{stackName: stackName, fullAgentImagePath: "gcr.io/datadoghq/agent:latest"})
+
+		// DogstatsD Standalone
+		suite.Run(t, &DockerSuite{stackName: stackName, fullAgentImagePath: "datadog/dogstatsd:latest"})
+	}
```

Comment:
Pulumi stack parameters can already be passed to the tests via the `-c` parameter.
So, we don’t need to introduce a new environment variable for that purpose.

Reply:
Yes makes sense, didn't know this existed with the invoke task!
Thanks